checks/core.py

```diff
@@ -134,5 +134,18 @@ def checkPortableMode(lines):
 
 def checkSafeMode(lines):
     if search('Safe Mode enabled.', lines):
-        return [LEVEL_WARNING, "Safe Mode Enabled",
-                """You are running OBS in Safe Mode. Safe Mode disables third-party plugins and prevents scripts from running."""]
+        modulesNotLoaded = search('not on safe list', lines)
+        moduleNames = []
+
+        for line in modulesNotLoaded:
+            found = re.search(r"'([^']+)'", line)
+            if found:
+                moduleNames.append(found.group(1))
```

Comment:
The raw analyzer output is a bit messy. Add newlines as appropriate? You don't need to add indentation. Ideally, we would have separate modes for output (raw CLI vs. HTML), but that's out of scope here, as other items already output HTML.
```
Warning:
Safe Mode Enabled (6)
    You are running OBS in Safe Mode. Safe Mode disables third-party plugins and
    prevents scripts from running. The following modules were not
    loaded:<br><ul><li>enc-amf</li><li>frontend-
    tools</li><li>logi_obs_plugin_x64</li><li>obs-websocket</li><li>vertical-
    canvas</li><li>win-capture-audio</li></ul>
```

Reply:
![image](https://github.com/obsproject/loganalyzer/assets/86465454/eba8bb37-508d-48e3-af89-663aa05ff588)

Updated to add some new lines